hugegraph-core/src/main/java/com/baidu/hugegraph/backend/tx/GraphIndexTransaction.java

```diff
@@ -268,6 +287,30 @@ protected void updateIndex(Id ilId, HugeElement element, boolean removed) {
         }
     }
 
+
+    private List<String> getAllCombinationList(List<String> prefixValues){
```

Comment:
not sure what is the intention of the method

Reply:
see concatValues comment